session/pingpong/invoice_payer.go

```diff
@@ -204,12 +192,16 @@ func (ip *InvoicePayer) recoverGrandTotalPromised() (uint64, error) {
 		return nil
 	}
 
-	err := backoff.Retry(toRetry, boff)
-	if err != nil {
-		log.Error().Err(err).Msg("could not get balance from accountant")
```

Comment:
Why to remove more detailed description here? We could say that "Failed after retry".

Reply:
It's moved one layer up